app/models/dream.rb

```diff
@@ -33,4 +40,21 @@ def keywords
     	puts 'Error in keyword extraction call: ' + response['statusInfo']
     end
   end
+
+  def keyword_analysis
+    alchemyapi = AlchemyAPI.new
+    response = alchemyapi.keywords('text', text, 'sentiment' => 1)
+    if response['status'] == 'OK'
+      keywords = response['keywords']
+      keywords.each do |keyword|
+        word = Keyword.find_or_create_by(text: keyword["text"])
+        keyword["mixed"].nil? ? mixed = false : mixed = true
+        relevance = keyword["relevance"]
+        sentiment = keyword["sentiment"]["score"]
+        DreamsKeyword.create(dream: self, keyword: word, relevance: relevance, sentiment: sentiment, mixed: mixed)
+      end
+    else
+    	puts 'Error in keyword extraction call: ' + response['statusInfo']
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Do not write to stdout. Use Rails' logger if you want to log.